front-end/src/components/Post/GovernanceSideBar/Proposals/SecondProposal.tsx

```diff
@@ -90,9 +90,9 @@ const SecondProposal = ({ className, proposalId, address, accounts, onAccountCha
 			{ noAccount
 				? <GetAccountsButton />
 				:loadingStatus.isLoading
-					? <div className={'LoaderWrapper'}>
+					? <Card className={'LoaderWrapper'}>
 						<Loader text={loadingStatus.message}/>
```

Comment:
I did not add a `timeout` here as `Loader` gets shown while the user has to sign via the extension. Same for `VoteReferendum` and `VoteMotion`